src/app/components/cards/TransferHistoryRow/story.jsx

```diff
@@ -0,0 +1,136 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withKnobs, date, select, number, text } from '@storybook/addon-knobs';
+
+import { IntlProvider } from 'react-intl';
+
+import { Provider } from 'react-redux';
+import { createStore } from 'redux';
+
+import rootReducer from 'app/redux/RootReducer';
+import { receiveState } from 'app/redux/GlobalReducer';
+import mockState from '../../../../../api_mockdata/get_state';
+
+import TransferHistoryRow from './';
+
+const store = createStore(rootReducer);
+
+// Load some fake blockchain state.
+// This component relies on many parts of the state tree - messy & hard to test!
+store.dispatch(receiveState(mockState));
```

Comment:
This is a good point of reference for working on other stories that will need store data.